crates/biome_html_analyze/tests/specs/a11y/noLabelWithoutControl/valid.html

```
@@ -0,0 +1,34 @@
+<!-- should not generate diagnostics -->
+
+<!-- Has `for` attribute and accessible text -->
+<label for="js_id">A label</label>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)